busylight/lights/hidlight.py

```diff
@@ -66,10 +66,10 @@ def udev_rules(cls, mode: int = 0o0666) -> List[str]:
         ]
 
         devices = cls.supported_device_ids()
-        rules.append(f"# Rules for {cls.vendor} Family of Devices: {len(devices)}")
+        rules.append(f"# Rules for {cls.vendor()} Family of Devices: {len(devices)}")
         for n, ((vid, pid), name) in enumerate(devices.items()):
             logger.info(f"udev rule for {vid:04x}, {pid:04x} {name}")
-            rules.append(f"# {n} {cls.vendor} {name}")
+            rules.append(f"# {n+1} {cls.vendor()} {name}")
```

Comment:
A more "pythonic" approach to solving the index problem is calling `enumerate` with the optional argument `start=N`. 

```python
for n, ((vid, pid), name) in enumerate(devices.items(), start=1):
    ...
```
I'll accept the PR and update with this change.

Reply:
Good call, I hadn't realized enumerate had that param. Thanks!